assets/js/manual-runner/ManualRunner.tsx

```diff
@@ -0,0 +1,575 @@
+import { MonacoEditor } from "#/monaco";
```

Comment:
I don't think we can call this `ManualRunner`. I'm being picky on language but this implies that the component is somehow responsible for runs. It implies it runs something.

I'd be happy with `ManualRunPanel`, since that's what it's been called in UX. But really I think this is an `InputBuilder` or `InputSelector`

Reply:
resolved. `ManualRunPanel`